zebra-scan/Cargo.toml

```diff
@@ -22,9 +22,10 @@ categories = ["cryptography::cryptocurrencies"]
 
 color-eyre = "0.6.2"
 indexmap = { version = "2.0.1", features = ["serde"] }
+itertools = "0.12.0"
 semver = "1.0.20"
 serde = { version = "1.0.193", features = ["serde_derive"] }
-tokio = "1.34.0"
+tokio = { version = "1.34.0", features = ["time"] }
```

Comment:
Yes, if the crate is built individually with no features, it needs access to the tokio time APIs.

Sometimes this works anyway in CI, I'm not sure why. Possibly because we're building all targets.

Reply:
This is now tested by #8042, which might fail the first time due to this bug in main